src/databases/Xolotl/avtXolotlFileFormat.C

```diff
@@ -161,6 +161,57 @@ avtXolotlFileFormat::GroupInfo(hid_t loc_id, const char *name, const H5L_info_t
 }
 
 
+// ****************************************************************************
+//  Method: avtXolotlFileFormat::PopulateNetworkGroupMetadata
+//
+//  Purpose:
+//      Gets the ntwork group meta data
+//
+//  Programmer: James Kress
+//  Creation:   February 19, 2021
+//
+// ****************************************************************************
+void
+avtXolotlFileFormat::PopulateNetworkGroupMetaData()
+{
+    // We need to stash the networkGroup attributes
+    hid_t normalSizeAttr = H5Aopen(networkGroup, "normalSize", H5P_DEFAULT);
+    if (normalSizeAttr < 0)
+    {
+        FreeUpResources();
+        EXCEPTION1(InvalidDBTypeException, "No 'normalSize' attribute.");
+    }
+
+    hid_t superSizeAttr = H5Aopen(networkGroup, "superSize", H5P_DEFAULT);
+    if (superSizeAttr < 0)
+    {
+        FreeUpResources();
+        EXCEPTION1(InvalidDBTypeException, "No 'superSize' attribute.");
+    }
+
+    int err2 = -1;
+    err2 = H5Aread(normalSizeAttr, H5T_NATIVE_INT, &normalSize);
+    if (err2 < 0)
+    {
+        FreeUpResources();
+        EXCEPTION1(InvalidDBTypeException, "cannot read 'normalSize' var.");
+    }
+    err2 = H5Aread(superSizeAttr, H5T_NATIVE_INT, &superSize);
+    if (err2 < 0)
+    {
+        FreeUpResources();
+        EXCEPTION1(InvalidDBTypeException, "cannot read 'superSize' var.");
+    }
+    if (debug) cerr << "normalSize="<<normalSize<<endl;
+    if (debug) cerr << "superSize="<<superSize<<endl;
```

Comment:
Printing to the terminal with debugging isn't how we normally do things. We usually just write to one of the debug logs, such as:

debug1 << "normalSize=" << normalSize << endl;
debug1 << "superSize=" << superSize << endl;

Reply:
Thanks, fixed.